R/data.R

```diff
@@ -57,14 +57,19 @@ spim_data <- function(date, region, model_type, rtm, serology,
 spim_data_single <- function(date, region, model_type, rtm, serology,
                              trim_deaths, trim_pillar2, full_data,
                              fit_to_variants, sircovid_model) {
+
+  pillar2_over25_age_bands <- c("25_49", "50_64", "65_79", "80_plus")
+  pillar2_age_bands <- c("under15", "15_24", pillar2_over25_age_bands)
```

Comment:
Perhaps now these are only needed in `spim_lancelot_data_rtm`, this could be moved back into there?

Reply:
Thanks, done